src/lib/entities/audio/specs/audio-instance.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { get } from 'svelte/store';
+import { src, currentTime, paused, move, pause, play, seek } from '..';
+import { duration } from '../model/audio-instance';
+
+describe('Audio playback API', () => {
+  const defaultSrc = 'http://commondatastorage.googleapis.com/codeskulptor-assets/Evillaugh.ogg';
+  const playDefault = () => play(defaultSrc);
+
+  it('loads audio', () => {
+    playDefault();
+    expect(get(src)).toEqual(defaultSrc);
+    expect(get(paused)).toBeFalsy();
+    pause();
+  });
+
+  it('pauses playback', () => {
+    playDefault();
+    pause();
+    play();
+    expect(get(paused)).toBeFalsy();
+    pause();
+    expect(get(paused)).toBeTruthy();
+  });
+
+  it('controls playback current time', async () => {
+    playDefault();
+    await new Promise((r) => setTimeout(r, 1000));
```

Comment:
issue: explicit hard-coded waits could lead to test flake and unnecessary slowdown of test execution

question: why do we need it? Can we mock stuff in order to remove the need for this wait?